libs/build_api_v2.py

```diff
@@ -87,8 +95,16 @@ def build_region_timeseries(
     for row in timeseries.yield_records():
         # Timeseries records don't have population
         row[CommonFields.POPULATION] = region_summary.population
-        actual = _build_actuals(row)
-        timeseries_row = ActualsTimeseriesRow(**actual.dict(), date=row[CommonFields.DATE])
+        actual = _build_actuals(row).dict()
+
+        # Don't include vaccinations in timeseries before first possible vaccination
+        # date to not bloat timeseries.
+        if row[CommonFields.DATE] < USA_VACCINATION_START_DATE:
```

Comment:
Why are we doing this by date instead of just checking for nulls or something?  This feels unnecessarily fragile (if we backfill the vaccine data we'll have to update this).

Reply:
That said, I don't care too strongly. 😁

Reply:
This is meant to actually handle backfilling - if we were to backfill, any data included after this date will be included (and this date is the first day of vaccination for the US, so there can't be vaccinations from any day earlier). 

If we were just to remove nulls, if a state or county didn't report for a day, (say in the middle of a timeseries) we would remove the field, which to me feels less consistent than saying "this field starts on december 14th"

Reply:
Ah, right on.  Makes sense! 👍